### PR TITLE
input: fix return key not always working

### PR DESCRIPTION
**Pull request by cgestes**
fixes https://github.com/memspace/zefyr/issues/234

### CODE REVIEW

**Comment by pulyaevskiy** [comment]
I'm having no luck reproducing this issue. Pressing 'return' key in the iOS simulator works as expected.

**Comment by cgestes** [comment]
Android simulator.

File: packages/zefyr/lib/src/widgets/input.dart [hunk 1]
```diff
@@ -97,6 +97,9 @@ class InputConnectionController implements TextInputClient {
   @override
   void performAction(TextInputAction action) {
     // no-op
```
Comment (britannio) [inline packages/zefyr/lib/src/widgets/input.dart:99]: is it still a no-op after this change?

**Comment by cgestes** [comment]
Core rewrite in branch 1.0-dev (see https://github.com/memspace/zefyr/issues/409) makes this PR irrelevant.

**Pull request closed without merging**